server/src/main/java/org/opensearch/index/query/MultiTermQueryBuilder.java

```diff
@@ -36,9 +36,4 @@
  *
  * @opensearch.internal
  */
-public interface MultiTermQueryBuilder extends QueryBuilder {
-    /**
-     * Get the field name for this query.
-     */
-    String fieldName();
-}
+public interface MultiTermQueryBuilder extends QueryBuilder, WithFieldName {}
```

Comment:
All the classes implementing `MultiTermQueryBuilder` are extending from `AbstractQueryBuilder`, hence already implementing `QueryBuilder`. `MultiTermQueryBuilder` looks weirdly designed, but something we can discuss as part of https://github.com/opensearch-project/OpenSearch/issues/15487